tests/test_csc.py

```diff
@@ -45,7 +45,7 @@ async def test_bin_script(self):
         """Test running from the command line script.
         """
         await self.check_bin_script(
-            name="Rotator",
+            name="MTRotator",
             index=None,
             exe_name="run_rotator.py",
```

Comment:
Don't forget to update this line when you rename the run_rotator.py script to run_mtrotator.py.